src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1402,18 +1402,16 @@ InferenceProfiler::SummarizeSendRequestRate(
   summary.send_request_rate = num_sent_requests / window_duration_s;
 }
 
-cb::Error
-InferenceProfiler::SummarizeServerStatsHelper(
+int64_t
+InferenceProfiler::DetermineStatsModelVersion(
     const cb::ModelIdentifier& model_identifier,
-    const std::map<cb::ModelIdentifier, cb::ModelStatistics>& start_status,
-    const std::map<cb::ModelIdentifier, cb::ModelStatistics>& end_status,
-    ServerSideStats* server_stats)
+    const std::map<cb::ModelIdentifier, cb::ModelStatistics>& stats)
 {
   // If model_version is an empty string then look in the end status to find
   // the latest (highest valued version) and use that as the version.
```

Comment:
Does this mean that if two versions are loaded and we only use the lower version, the stats for the highest version might be printed? For example, the composing model A has versions 2 and 3 and only version 2 is being used but we would print the stats for version 3?